SQF/dayz_code/stringtable.xml

```diff
@@ -7475,7 +7475,7 @@
 		</Key>
 		<Key ID="str_death_shothead">
 			<English>a gunshot to the head.</English>
-			<German>durch einen Schuss in den Kopf.</German>
+			<German>durch einen Kopfschuss.</German>
```

Comment:
Yes this one. It sounds a bit dumpy. It would be the same for the english one. "Died from a headshot." It sounds just not that good. Better is the string we already have  "Died from a gunshot to the head."